### ISSUE TITLE
Servicemember iconography set

### ISSUE THREAD

**Issue by elizbond** [cfpb/design-manual#38]
At the CFPB, we have three different levels of iconography and illustration that we use throughout our web and print experiences. In the fall of 2013, the need for an iconography set to represent the Office of Servicemembers Affairs became apparent. The current state of the set is listed below.
## Illustration

![screen shot 2014-03-11 at 11 07 52 am](https://f.cloud.github.com/assets/6798337/2386761/aee5ccd6-a92f-11e3-84bd-6e8e3f1aaab1.png)
## Isoscon

![screen shot 2014-03-11 at 11 14 04 am](https://f.cloud.github.com/assets/6798337/2386780/cd50e0fc-a92f-11e3-91e1-5db86e67d626.png)
Business owner feedback is still pending, are there any revisions?
## Minicon

![screen shot 2014-03-11 at 11 14 48 am](https://f.cloud.github.com/assets/6798337/2386794/eda73ffe-a92f-11e3-9490-121f78672a52.png)
Current feedback - try a smaller stroke weight in the interior of the dogtag.

**Comment by elizbond** [comment]
Isocon - Has there been an update? @skennedy
Minicon - Can we try it with a chain? The overlap should be updated. Let's try it with context. @Dnpizarro

**Comment by Dnpizarro** [comment]
Below is the revised servicemember minicon. I provided 4 options: with the beaded chain and without, fill and no-fill (stroke). I kept the no chain option incase we felt the chain option felt too small. I dropped the stroke weight slightly. I also placed the minicons into buttons to give us a sense of context. 

![servicemember_minicons](https://f.cloud.github.com/assets/6852363/2398466/cf81861c-a9f5-11e3-9e6b-2ee51676569f.png)

![servicemember_buttons](https://f.cloud.github.com/assets/6852363/2398475/d81efdcc-a9f5-11e3-8903-d5715f76ae92.png)
